_sass/page.scss

```diff
@@ -27,6 +27,10 @@ body {
     height: auto;
 }
 
+.banner .logo img {
+    height: 40px;
+}
+
```

Comment:
Is this responsive?

Reply:
I checked it and it was looking fine for all displays, but I will go ahead and make this responsive using em instead of px.

Reply:
Made the required changes, removed the configure changes, and made the logo responsive.